src/Services/GTT/GTT.Api/GroupManagement/GetGroups.cs

```diff
@@ -0,0 +1,74 @@
+using System.Net;
+using FluentValidation;
+using GTT.Api.Configuration;
+using GTT.Application.Extensions;
+using GTT.Application.Queries;
+using GTT.Application.Response;
+using GTT_API.ExerciseLibraryManagement;
+using MediatR;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json;
+
+namespace GTT_API.GroupManagement
+{
+    public class GetGroups
+    {
+        #region Private Members
+        private readonly ILogger _logger;
+        private readonly IMediator _mediator;
+        #endregion
+
+        #region Constructors
+        public GetGroups(ILoggerFactory loggerFactory, IMediator mediator)
+        {
+            _logger = loggerFactory.CreateLogger<GetGroups>();
+            _mediator = mediator;
+        }
+        #endregion
+
+        [Function("GetGroup")]
+        [OpenApiOperation(nameof(GetGroups), "Groups")]
+        [OpenApiParameter("PageSize", In = ParameterLocation.Query, Required = false, Type = typeof(int))]
+        [OpenApiParameter("PageIndex", In = ParameterLocation.Query, Required = false, Type = typeof(int))]
+        [OpenApiParameter("Keyword", In = ParameterLocation.Query, Required = false, Type = typeof(string))]
```

Comment:
We should create a separated API for SearchGroups, so remove Keywork parameter for this API.

Or 

If we keep this API for SearchGroup, then create new API for GetGroups